db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/StatementSchemaPermission.java

```diff
@@ -337,4 +360,35 @@ public String getSchemaName() {
 		return schemaName;
 	}
 
+	/**
+	 * Routine to check if another instance of access descriptor matches this.
+	 * Used to ensure only one access descriptor for a table of given privilege is created.
+	 * Otherwise, every column reference from a table may create a descriptor for that table.
+	 *
+	 * @param obj	Another instance of StatementPermission
```

Comment:
nit: Remove as adds nothing (we don't know whether it's a `StatementPermission` or not)

Reply:
done.